promql/testcases/expand.go

```diff
@@ -124,6 +124,10 @@ func applyQueryTweaks(tc *comparer.TestCase, tweaks []*config.QueryTweak) *compa
 			resTC.Start = resTC.Start.Truncate(resTC.Resolution)
 			resTC.End = resTC.End.Truncate(resTC.Resolution)
 		}
+		if ms := time.Duration(t.AlignTimestampsToMS) * time.Millisecond; ms != 0 {
```

Comment:
Let's call it `d` here as above, since the `time.Duration` isn't in any specific unit like `ms` anymore, it's just a Go duration type.